uniflow-core/src/main/kotlin/io/uniflow/core/flow/DataFlow.kt

```diff
@@ -131,15 +119,25 @@ interface DataFlow {
      * Execute the action & catch any flowError
      * @param action
      */
+//    fun onAction(action: StateAction) {
```

Comment:
Why we still keeping this ?

Reply:
It's the old `onAction`, that was working without Actor under the hood